packages/core/src/internal/index.ts

```diff
@@ -4,10 +4,10 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-/** @internal private module to Clarity Core */
-
+import '@cds/core/polyfills';
```

Comment:
Most of our other imports point to a specific `.js` file. Should this one do that too?

`import '@cds/core/polyfills/index.js` vs. `import '@cds/core/polyfills'`?

Reply:
`.js` is required for relative imports or imports that by pass an entrypoint module (we typically avoid but `register.js` would be an example exception). This is mainly to prevent users from reaching into modules that are [intended to be private](https://unpkg.com/@cds/core@5.6.0/package.json). I made polyfills an entrypoint to ensure its side effect of running the polyfill code and not getting tree-shaken away by bundlers. So now in the generate package.json it will have these entries added,

```json
"sideEffects": [
    "./polyfills/index.js" 
],
"exports": {
    "./polyfills": "./polyfills/index.js",
}
```